update-resources.sh

```diff
@@ -40,171 +40,173 @@ function chkfs() {
+  wget -q -O ./windows/plink64.exe "https://the.earth.li/~sgtatham/putty/latest/w64/plink.exe"
 }
 
 function add_deepce() {
   info "Downloading deepce"
   chkfs "./linux/"
-  wget -nv -O ./linux/deepce.sh "https://github.com/stealthcopter/deepce/raw/master/deepce.sh"
+  wget -q -O ./linux/deepce.sh "https://github.com/stealthcopter/deepce/raw/master/deepce.sh"
 }
 
 function add_webshells() {
   info "Downloading webshells"
   chkfs "./webshells/PHP/wso-webshell/"
-  wget -nv -O ./webshells/PHP/wso-webshell/wso.php "https://raw.githubusercontent.com/mIcHyAmRaNe/wso-webshell/master/wso.php"
-  sed -i 's/fa769dac7a0a94ee47d8ebe021eaba9e/0fc3bcf177377d328c77b2b51b7f3c9b/g' ./webshells/PHP/wso-webshell/wso.php
+  wget -q -O ./webshells/PHP/wso-webshell/wso.php "https://raw.githubusercontent.com/mIcHyAmRaNe/wso-webshell/master/wso.php"
+  perl -i -pe's/fa769dac7a0a94ee47d8ebe021eaba9e/0fc3bcf177377d328c77b2b51b7f3c9b/g' ./webshells/PHP/wso-webshell/wso.php
   echo 'exegol4thewin' > ./webshells/PHP/wso-webshell/password.txt
   # Setting password to exegol4thewin
   chkfs "./webshells/ASPX"
-  wget -nv -O ./webshells/ASPX/webshell.aspx "https://raw.githubusercontent.com/xl7dev/WebShell/master/Aspx/ASPX%20Shell.aspx"
+  wget -q -O ./webshells/ASPX/webshell.aspx "https://raw.githubusercontent.com/xl7dev/WebShell/master/Aspx/ASPX%20Shell.aspx"
 }
 
 function add_ysoserial_net() {
   info "Downloading ysoserial"
   chkfs "./windows/"
-  url=$(curl -s https://github.com/pwntester/ysoserial.net/releases/latest | grep -o '"[^"]*"' | tr -d '"' | sed 's/tag/download/')
-  tag=${url##*/}
-  prefix=${tag:1}
-  wget -nv -O ./ysoserial_net.zip "$url/ysoserial-$prefix.zip"
-  unzip -d ./windows/ ./ysoserial_net.zip
-  rm -r ./windows/ysoserial.net
+  url=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/pwntester/ysoserial.net/releases/latest)
+  version=${url##*v}
+  wget -q -O ./ysoserial_net.zip "https://github.com/pwntester/ysoserial.net/releases/latest/download/ysoserial-$version.zip"
+  unzip -q -d ./windows/ ./ysoserial_net.zip
+  rm -rf ./windows/ysoserial.net
   mv ./windows/Release/ ./windows/ysoserial.net
   rm ./ysoserial_net.zip
 }
 
 function add_http-put-server() {
   info "Downloading http-put-server for Python3"
   chkfs "./linux/"
-  wget -nv -O ./linux/http-put-server.py https://gist.githubusercontent.com/mildred/67d22d7289ae8f16cae7/raw/214c213c9415da18a471d1ed04660022cce059ef/server.py
+  wget -q -O ./linux/http-put-server.py "https://gist.githubusercontent.com/mildred/67d22d7289ae8f16cae7/raw/214c213c9415da18a471d1ed04660022cce059ef/server.py"
 }
 
 function add_azurehound() {
   info "Downloading AzureHound"
   chkfs "./windows/"
-  wget -nv -O ./windows/AzureHound.ps1 "https://raw.githubusercontent.com/BloodHoundAD/BloodHound/master/Collectors/AzureHound.ps1"
+  wget -q -O ./windows/AzureHound.ps1 "https://raw.githubusercontent.com/BloodHoundAD/BloodHound/45c04e7292a44362af1031cc0d7bd3748433ac4d/Collectors/AzureHound.ps1"
```

Comment:
For AzureHound, we should get the latest release from the new repository instead of the last release on the BloodHound one.

Reply:
Agreed

Reply:
New releases of AzureHound are binaries, instead of a ps1 as before.
It is still ok to take the last releases ?

Reply:
Indeed, it would be better to install AzureHound in exegol-images? So there is no real reason now to upload the ps1 to a windows machine ?

Reply:
@ShutdownRepo what do you think ?

Reply:
Agreed, let's remove the AzureHound.ps1 pull